donate/payments/views.py

```diff
@@ -797,7 +801,9 @@ def form_valid(self, form, send_data_to_basket=True):
                     exc_info=True
                 )
 
-        elif send_data_to_basket:
+        if send_data_to_basket:
```

Comment:
Changed the `elif` to an `if`. This will affect Thunderbird. We are now sending subscription info from thunderbird-donate _and_ donate-wagtail to Basket.

Reply:
These options are all independent of each other for now.
I think the cleanest solution will be:
* Add an environment variable DONATE_NEWSLETTER_SUBSCRIBE_METHOD for donate staging and prod and thunderbird staging and prod
* Currently for donate:
    *  Staging:  DONATE_NEWSLETTER_SUBSCRIBE_METHOD = BASKET
    * Prod:  DONATE_NEWSLETTER_SUBSCRIBE_METHOD = SQS
* Currently for Thunderbird:
    * Staging:  DONATE_NEWSLETTER_SUBSCRIBE_METHOD = MAILCHIMP
    * Prod:  DONATE_NEWSLETTER_SUBSCRIBE_METHOD = MAILCHIMP

  Then for the logic
```python
if DONATE_NEWSLETTER_SUBSCRIBE_METHOD == "BASKET":
    # basket HTTP API code here
if DONATE_NEWSLETTER_SUBSCRIBE_METHOD == "SQS":
    # sqs code here
if DONATE_NEWSLETTER_SUBSCRIBE_METHOD == "MAILCHIMP":
    # mailchimp code here
```